arrow/arrow.py

```diff
@@ -616,6 +661,11 @@ def replace(self, **kwargs):
 
             if key in self._ATTRS:
                 absolute_kwargs[key] = value
+            elif key == "fold":
+                raise AttributeError(
+                    "Setting fold attribute using replace() is not supported, set directly "
```

Comment:
You should definitely support `fold` in `replace`, and as I mention in my other comment, it should in fact be the *only* way to set the fold.